src/tests/functional/plugin/shared/include/base/ov_behavior_test_utils.hpp

```diff
@@ -62,8 +62,8 @@ class APIBaseTest : public CommonTestUtils::TestsCommon {
 
     void SetUp() override {
         set_api_entity();
-        auto test_name = this->GetTestName();
-        k = test_name.find("_mandatory") != std::string::npos || test_name.find("mandatory_") != std::string::npos ? 1 : 0;
+        auto suite_name = this->GetSuiteName();
```

Comment:
test_name should contain test_suite name

Reply:
no, for test 
TEST(IEClassBasicTest, smoke_registerNonExistingPluginFileThrows)

test_name will contain only smoke_registerNonExistingPluginFileThrows , suite_name will contains instigation prefix too

Reply:
And what about test name? We can set prefix for test_name also..

Reply:
yes, you are right ! updated